pelops/datasets/veri.py

```diff
@@ -86,10 +121,14 @@ def __create_chip(self, img_dir, img_name):
         splitter = img_name.split("_")
         misc = dict()
 
-        filepath = img_dir + "/" + img_name
+        filepath = os.path.join(img_dir, img_name)
         car_id = int(splitter[0])
         cam_id = int(utils.get_numeric(splitter[1]))
         time = datetime.datetime.fromtimestamp(int(splitter[2]))
         misc["binary"] = int(os.path.splitext(splitter[3])[0])
 
+        color, vehicle_type = self.__color_type.get(car_id, (None, None))
+        misc["color"] = color
+        misc["vehicle_type"] = vehicle_type
+
```

Comment:
There are vehicles without color and type, mainly the entire test set. Give that that is the case, I think it is reasonable to fill in `None` as a default and let the user check.